cluster-soak/kubernetes/rbac.yml

```diff
@@ -19,24 +19,4 @@ roleRef:
   kind: Role
   name: pod-reader
   apiGroup: rbac.authorization.k8s.io
----
-kind: Role
-apiVersion: rbac.authorization.k8s.io/v1
-metadata:
-  name: lease-access
-rules:
-  - apiGroups: ["akka.io"]
-    resources: ["leases"]
-    verbs: ["get", "create", "update", "delete", "list"]
----
-kind: RoleBinding
-apiVersion: rbac.authorization.k8s.io/v1
-metadata:
-  name: lease-access
-subjects:
-  - kind: User
-    name: system:serviceaccount:akka-long-running:default
-roleRef:
-  kind: Role
-  name: lease-access
-  apiGroup: rbac.authorization.k8s.io
+
```

Comment:
Role/rolebinding no longer necessary? or taken from another file now?

Reply:
The role/role binding removed were for the lease which isn't used in the soak test